store/postgres/tests/relational.rs

```diff
@@ -197,8 +202,13 @@ fn update_entity(conn: &PgConnection, layout: &Layout, entity_type: &str, entity
         entity_type.to_owned(),
         entity.id().unwrap(),
     );
-    let errmsg = format!("Failed to update entity {}[{}]", entity_type, key.entity_id);
-    layout.update(&conn, &key, entity, 1).expect(&errmsg);
+    let entity_id = key.entity_id.clone();
+    let entity_type = EntityType::from(entity_type);
+    let mut entities = vec![(key, entity)];
+    let errmsg = format!("Failed to insert entity {}[{}]", entity_type, entity_id);
+    layout
+        .update(&conn, &entity_type, &mut entities, 0)
+        .expect(&errmsg);
```

Comment:
Similarly here - it would also be good to test that with more than one entity. Since creating entities is annoying from Rust code, there's an `entity!` macro that can ease notation a bit.